### PR TITLE
Improve debug output

### PR DESCRIPTION
**Pull request by mlangc**
These improvements to the debug output have been implemented in connection with https://github.com/scala-ide/scala-ide/pull/1106, but they should be generally useful.

### CODE REVIEW

**Comment by mlangc** [comment]
Now this is very strange, since I didn't change a thing but debug output, and all tests pass locally.

**Comment by mlangc** [comment]
Retest this please

File: src/test/java/scala/tools/refactoring/common/TracingHelpersTest.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package scala.tools.refactoring.common
+
+import org.junit.Test
+import org.junit.Assert._
+
+
+class TracingHelpersTest {
```
Comment (sschaef) [inline src/test/java/scala/tools/refactoring/common/TracingHelpersTest.scala]: Don't you want to enable this test in `RefactoringTestSuite`?

Comment (mlangc) [inline src/test/java/scala/tools/refactoring/common/TracingHelpersTest.scala]: Of course, thanks for spotting!

**Comment by sschaef** [comment]
LGTM

**Pull request merged**